modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,227 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-bonded-sriov-network_{context}"]
+= Configure sysctl on a bonded SR-IOV network 
+
+As a cluster administrator, set the interface specific sysctls on a bonded interface created from two SR-IOV interfaces by adding the tuning configuration to the optional `Plugins` parameter of the bond network attachment definition.
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` custom resources that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+To change specific interface-level network sysctls create the SriovNetwork custom resource (CR)  with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the SriovNetwork custom resource (CR) for the bonded interface as in the following example CR. Save the YAML as the file `sriov-network-attachment.yaml`. 
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1 
+kind: SriovNetwork 
+metadata: 
+  name: allvalidflags 
+  namespace: openshift-sriov-network-operator 
+spec: 
+  resourceName: policyallflags 
+  networkNamespace: sysctl-tuning-test 
+  capabilities: '{ "mac": true, "ips": true }'
+----
+
+. Create the SriovNetwork resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-bond-network-interface.yaml
```

Comment:
The name of the file that you suggested to add is `sriov-network-attachment.yaml` and here you use another filename.

Reply:
corrected